app/templates/index.html

```diff
@@ -90,17 +89,13 @@ <h1>'Allo, 'Allo!</h1>
       r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
       ga('create','UA-XXXXX-X');ga('send','pageview');
     </script>
-
-    <!-- build:js scripts/vendor.js -->
-    <!-- bower:js -->
-    <!-- endbower -->
-    <!-- endbuild -->
+    <!-- build:js scripts/vendor.js -->	
+    <!-- endbuild -->	
+    <% if (includeJQuery || includeBootstrap) { %>
+      <script type="text/javascript" src="/node_modules/jquery/dist/jquery.min.js"></script>
+    <% } %>
     <% if (includeBootstrap) { %>
-    <!-- build:js scripts/plugins.js -->
-<% bsPlugins.forEach(function (plugin) { -%>
-    <script src="<%= bsPath + plugin %>.js"></script>
-<% }) -%>
-    <!-- endbuild -->
+      <script type="text/javascript" src="/node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
```

Comment:
I think it's best to place both jQuery and Bootstrap under the `vendor.js` build comment.

Reply:
I agree! :-)